shoes-core/lib/shoes/color.rb

```diff
@@ -24,7 +24,7 @@ def initialize(*args) # red, green, blue, alpha = OPAQUE)
   - #{self.class}.new(red, green, blue)
   - #{self.class}.new(red, green, blue, alpha)
 EOS
-        fail ArgumentError, message
+        raise ArgumentError, message
```

Comment:
jeez rubocop reversing its opinion on fail... one of the few things I learned from it and liked as a distinction. :|